packages/components/src/Inject/Inject.md

```diff
@@ -107,6 +107,34 @@ function MyUse() {
 
 In this case Action will not be wrapped but WhatEver will be.
 
+# How to support injectable props (Inject.getReactElement)
+
+injectable props are props that can have three types:
+* react valid element
```

Comment:
in cas of a valid react element isn't getComponent superfluous ?

Reply:
yes it will not be used in this case